lib/led.js

```diff
@@ -1,9 +1,129 @@
 var IS_TEST_MODE = global.IS_TEST_MODE || false;
 var Board = require("../lib/board.js");
+var __ = require("../lib/fn.js");
+var nanosleep = require("../lib/sleep.js").nano;
 var Pins = Board.Pins;
 
 var priv = new Map();
 
+var Controllers = {
+  PCA9685: {
+    COMMANDS: {
+      value: {
+        PCA9685_MODE1: 0x0,
+        PCA9685_PRESCALE: 0xFE,
+        LED0_ON_L: 0x6
+      }
+    },
+    initialize: {
+      value: function(opts) {
+        this.address = opts.address || 0x40;
+        this.pwmRange = opts.pwmRange || [0, 4095];
+
+        if (!this.board.Drivers[opts.address]) {
+          this.io.sendI2CConfig();
+          this.board.Drivers[opts.address] = {
+            initialized: false
+          };
+
+          // Reset
+          this.io.sendI2CWriteRequest(this.address, [this.COMMANDS.PCA9685_MODE1, 0x0]);
```

Comment:
Update these i2c calls to the new API: https://github.com/jgautier/firmata#i2c (these should be `i2cWriteReg(address, reg, value)`)